runtime/handler.go

```diff
@@ -131,3 +132,22 @@ func uniqueSubjects(in []sparql.Triple) []string {
 
 	return u
 }
+
+func removeDuplicates(elements []string) []string {
+	// Use map to record duplicates as we find them.
+	encountered := map[string]bool{}
+	result := []string{}
+
+	for v := range elements {
+		if encountered[elements[v]] == true {
```

Comment:
Do you need to have the `== true` part here?  Isn't this already a boolean?
If this is the case can you just `continue` and then you don't need the else clause.

Reply:
Ah, good catch, this was a copy-pasta, so I can clean it up quickly.

Reply:
How large is `elements` generally? I'm wondering about the performance of this dup removal technique (and not sure what technique in golang is most idiomatic).

Reply:
I might like to do a bit of benchmarking around this if folks think it'd be worth it. Else, meh.